example/app/package.json

```diff
@@ -18,22 +18,22 @@
     "@react-navigation/stack": "^6.0.11",
     "faker": "^4.1.0",
     "nanoid": "^3.3.3",
-    "react": "17.0.2",
-    "react-native": "0.68.1",
-    "react-native-gesture-handler": "~2.8.0",
+    "react": "18.2.0",
+    "react-native": "0.71.3",
+    "react-native-gesture-handler": "~2.9.0",
     "react-native-maps": "^0.30.1",
     "react-native-pager-view": "^5.4.24",
-    "react-native-reanimated": "~2.12.0",
+    "react-native-reanimated": "~2.14.4",
```

Comment:
Hi 👋, @gorhom  I have a question
Isn't this PR is for updating reanimated v3?
I'm wondering why the changed version is v2.14.4 of it.

Reply:
Yes, why is this using reanimated 2.14.4?